AR_LaTeX_homework.tex

```diff
@@ -0,0 +1,157 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tipa}
+\usepackage{qtree}
+\usepackage{phonrule}
+\usepackage{amsmath}
+\usepackage{graphicx}
+\graphicspath{ {./pics/} }
+\usepackage{wrapfig}
+\usepackage{CJKutf8}
+
+\title{LaTeX homework}
+\author{Agnieszka Rachlewicz }
+\date{May 2019}
+
+\begin{document}
+
+\maketitle
+
+\section{Section 1}
+\begin{enumerate}
+  \item Neurology (brain possibilities).
+  \item Phonology in general.
+  \item Somnology (about dreaming in general, includes lucid dreaming).
+\end{enumerate}
+\section{Section 2}
+
+% aɪm ən ˈɔːl.məʊst ˈtwɛn.ti-faɪv jɪr əʊld ˈstjuː.dənt ɒv ˈɪŋ.ɡlɪʃ fɪˈlɒl.ə.dʒi | maɪˌspeʃ.əl.aɪˈzeɪ.ʃən  ɪz ˈnætʃ.ər.əl  ˈlæŋ.ɡwɪdʒ ˈprəʊ.ses.ɪŋ  |   aɪm ɪn ðə ˈmɪdl ɒv maɪ ˌsek.ənd.dɪˈɡriː ænd maɪ ˈɪn.trəsts   ɑː  ˈlɜː.nɪŋ ˌdʒæp.ənˈiːz |  dɑːnsing   dʒæz  and   ˈfɪɡ.ə ˌskeɪ.tɪŋ wɪtʃ  ɪnˈkluːdz on ˈaɪs ˌskeɪ.tɪŋ | ˈfɪɡ.ə  ˈrəʊ.lə.skeɪ.tɪŋ and off-ice ˈek.sə.saɪzIz/
+English.\newline 
+
+\textipa{/aIm @n "O:l.m@Ust "twEn.ti faIv jI@ @Uld "stju:.d@nt @v "IN.glIS fI"l6l.@\texttoptiebar{\textdyoghlig}i || maI ""spES.@l.aI"zeI.S@n Iz "n\ae \texttoptiebar{\textteshlig}.@r.@l "l\ae \ng.gwI\texttoptiebar{\textdyoghlig} \space "pr@U.sEs.I\ng \space || aIm In \dh @ "mId.l @v maI ""sEk.@nd.dI"gri: | @nd maI "In.tr@sts \textscripta: | "l3:.nI\ng \space ""\texttoptiebar{\textdyoghlig}\ae p.@n"i:z | d\textscripta:nsI\ng \space \texttoptiebar{\textdyoghlig}\ae z | @nd  
+"fIg.@ ""skeI.tI\ng \space wI\texttoptiebar{\textteshlig} In"klu:\texttoptiebar{dz} | 6n aIs ""skeI.tI\ng \space | "fIg.@ "r@U.l@.skeI.tI\ng \space | @nd 6f aIs "ek.s@.saIzIz/}
+\newline \newline
+Polski.
+% Jestem dwudziestoczteroletnią studentką filologii angielskiej, na specjalizacji przetwarzanie języka naturalnego. 
+%Moimi zainteresowaniami są nauka języka japońskiego, tańczenie dżezu oraz jazda figurowa, która obejmuje jazdę na łyżwach, jazdę na łyżworolkach figurowych oraz imitacje czyli ćwiczenia poza lodem.
+\newline
+\newline
+\textipa{/jEstEm dvu\textdctzlig est\textopeno \texttoptiebar{\textteshlig}ter\textopeno let\textltailn  \~\textopeno \space studentk\~\textopeno \space fil\textopeno l\textopeno g\textsuperscript{j}i \~a\ng g\textsuperscript{j}Elsk\textsuperscript{j}ej | na spEcjal\textsuperscript{j}izac\textsuperscript{j}i pSEtfa\textyogh an\textsuperscript{j}e j\~E\~wz\textbari ka naturalnEg\textopeno \space || m\textopeno \textsuperscript{j}imi zaintErEs\textopeno van\textsuperscript{j}ami s\~\textopeno \space | nauka j\~Ez\textbari ka naturalnEg\textopeno \space | t\~a\~j\texttoptiebar{\textteshlig}En\textsuperscript{j}E d\texttoptiebar{\textyogh}Ezu \textopeno raz jazda figur\textopeno va | ktura obEjmujE jazdE na w\textbari\textyogh vax | jazdE na w\textbari\textyogh v\textopeno r\textopeno lkax figur\textopeno v1x | \textopeno raz imitacjE | \texttoptiebar{\textteshlig}1li t\texttoptiebar{\textctc}fi\textteshlig En\textsuperscript{j}a p\textopeno za l\textopeno dEm/}
```

Comment:
"nauka języka naturalnego"?
Why no word stress here?
Polish <c> = /ts/ (not /c/)!